gammapy/datasets/map.py

```diff
@@ -204,6 +208,7 @@ def __init__(
         gti=None,
         meta_table=None,
         name=None,
+        livetime_map=None,
```

Comment:
Please rename to `livetime`, the other attributes do not carry the `_map` prefix either. The type can be clarified in the docstring.